test/core/util/one_corpus_entry_fuzzer.cc

```diff
@@ -29,7 +29,7 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size);
 extern bool squelch;
 extern bool leak_check;
 
-int main(int argc, char** argv) {
+int main(int /*argc*/, char** argv) {
```

Comment:
Instead of ignoring argc, would it make sense to to assert that argc > 1 as we're using argv[1] below?

Reply:
Whoa, fuzzing the fuzzer....